### ISSUE TITLE
Feature Request: Bypass Watchdog if NS has recent BG

### ISSUE THREAD

**Issue by juehv** [xdrip-js/Logger#213]
I'm using the read-only mode of Logger to offline loop. However at home the loop is online and can read the BG values from NS (where xdrip on the phone is pushing to). As I don't carry the loop with me all the time, it reboots quite often which and drains the battery but is not nessecary. Would be nice if the watchdog would check the monitor/glucose.json if recent values are available and prevent the reboot if so (and thus only reboots if no recent values are available).

**Comment by scottleibrand** [comment]
This would need to be an optional feature IMO. If Logger is broken, I wouldn’t want to delay fixing it just because my other redundant glucose source is still available.

**Comment by juehv** [comment]
Yes I agree, but shouldn't be an issue to add another option in the xdripjs.json file
